webapp/channels/src/components/modal_controller/modal_controller.tsx

```diff
@@ -32,39 +32,39 @@ type Props = {
     };
 }
 
-export default class ModalController extends React.PureComponent<Props> {
-    public render(): React.ReactNode {
-        const {modals, ...props} = this.props;
-        const {modalState} = modals;
+const ModalController = ({
+    modals,
+    actions,
+}: Props) => {
+    const {modalState} = modals;
 
-        if (!modals) {
-            return null;
-        }
+    if (!modals) {
+        return null;
+    }
```

Comment:
I know the original file was this way, but this doesn't make much sense. If `!modals`, the previous statement would have generated an exception (cannot destructure undefined or bool).

Switch the two checks.